reactors/fcs-tasbe/src/create_color_model_object.m

```diff
@@ -1,11 +1,13 @@
-function [cm] = create_color_model_object(bead_file,blank_file,bead_model,bead_batch,translation_plot,noise_plot, fitc_channel_name,channel_mins)
+function [cm] = create_color_model_object(bead_file,blank_file,bead_model,bead_batch,translation_plot,noise_plot, fitc_channel_name,channel_mins,min_valid_count,pem_drop_threshold)
```

Comment:
This is not going to scale --- it needs to change to take a preferences object at some point (though not necessarily now).